### DISCUSSION TITLE
Chimes only work after a second door bell press..

### DISCUSSION THREAD

**Discussion by vadalus911** [dgreif/ring#660] [Q&A]
Hi Guys,

Dont know if anyone has seen this before but i use a UnifiG4 doorbell to trigger my Ring Chimes. HomeKit receives the doorbell trigger but only activates the Ring Chimes when i press the door bell twice. Homekit registers two presses when i do this. If i test the automation, it always works and sounds the chimes.

Is there a possible issue where the Chimes 'fall alseep'? I know i am going to a remote server to trigger these chimes so it's not ideal..

I do not see any logging from the Rung plugin at all when i sound the Chimes so i am at a loss to debug any further

**Comment by dgreif** [comment]
Do you have the automation set up for a single press or double press on the UniFi camera? It might support both, so verify you are using the right trigger. I'm not aware of any issues with the chime going to sleep

**Reply by vadalus911** [reply]
if definitly works with a double press! but also sometimes works with a single press. i did not see anything in the unifi homebridge documentation about a double press. I only have a single G4 doorbell trigger exposed in Homekit so have not seen anything about a double press there... is there a link you coujld point me to?

**Reply by dgreif** [reply]
This is what I mean when I say it might support both.  With some switches, you can register actions for single/double/triple presses: 
![image](https://user-images.githubusercontent.com/3026298/121409139-fe20f100-c915-11eb-82a9-10afa864ac9d.png)

**Reply by dgreif** [reply]
@vadalus911 did you ever get this issue figured out?

**Comment by vadalus911** [comment]
Hey

I managed to change the automated to ring twice which solves a lot of the
issues but by no means 100% I’m afraid

Liam
